packages/iTwinUI-react/src/core/ButtonGroup/ButtonGroup.tsx

```diff
@@ -110,7 +110,7 @@ export const ButtonGroup = React.forwardRef<HTMLDivElement, ButtonGroupProps>(
             )}
 
           {visibleCount < items.length
-            ? items.slice(0, visibleCount - 1)
+            ? items.slice(0, Math.max(0, visibleCount - 1))
```

Comment:
Yes, I think that is what Vilius meant. I also tested it and it worked.

Reply:
Thanks!